string task
geometry_msgs/PoseStamped head
---
geometry_msgs/PoseStamped base_goal

